akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/PersistentActorCompileOnlyTest.scala

```diff
@@ -303,12 +315,12 @@ object PersistentActorCompileOnlyTest {
                 case GotMetaData(data) ⇒
                   basket = basket.updatedWith(data)
                   if (isFullyHydrated(basket, state)) {
-                    stash.foreach(ctx.self ! _)
-                    stash = Nil
+                    Effect.unstash(buffer.unstashAll())
```

Comment:
How about `unstash(buffer)`? `unstashAll` is not that super useful since most messages end in a persist, so there will be a lot of "stash everything", "oh wait, unstash everything" loops